core/src/main/scala/org/apache/spark/internal/config/UI.scala

```diff
@@ -203,4 +203,15 @@ private[spark] object UI {
     .stringConf
     .transform(_.toUpperCase(Locale.ROOT))
     .createWithDefault("LOCAL")
+
+  val ENABLED_STREAMING_UI_CUSTOM_METRIC_LIST =
+    ConfigBuilder("spark.ui.sql.streaming.enabledCustomMetricList")
+      .internal()
+      .doc("Configures a list of custom metrics on Structured Streaming UI, which are enabled. " +
+        "The list contains the name of the custom metrics separated by comma. In aggregation" +
+        "only sum used.")
+      .version("3.1.0")
+      .stringConf
+      .toSequence
+      .createWithDefault(Nil)
```

Comment:
Do we need to add some common custom metrics here by default?

Reply:
We've discussed w/ @HeartSaVioR that by default it must be empty and in need can be added.

Reply:
The main reason was that these are full custom so any default is hard to come up with.
If anybody thinks we can add defaults, please add comments and name them.